src/uniswap-summaries.md

```diff
@@ -3280,6 +3280,13 @@ module SOLIDITY-MATHSQRT-SUMMARY
        <call-stack>... .List => ListItem(frame(K, E, FUNC)) </call-stack>
     requires V >uMInt 3p256 [priority(40)]
 
+  // While loop rewrite and condition evaluation
+  rule <k> while (( x < z ) #as Cond) Body:Statement Ss:Statements => if (v(Vx <uMInt Vz, bool)) {Body while(Cond) Body} else {.Statements} ~> Ss ...</k>
+       <summarize> true </summarize>
```

Comment:
Same question from the case `y <= 3 && y != 0` about the cell variables

Reply:
Here, we are already within the body of the function `mathSqrt`, i.e. to get to this rule the one doing the matching and changes to the configuration related to the call itself has already matched and done its rewriting. So, we don't need to match any of those cells.